pages/getting-started/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: Getting Started
+---
+# New Laptop Setup
+
+## MacBook
+Be sure to turn on disk encryption on the initial setup of the laptop!
+
+You will probably need:
+
+ - Homebrew (Go to http://brew.sh/ and follow the install instructions)
+ - A shell of your choice
+ - Your favorite Editor
+
+Eventually you will also need (add these on a project by project basis)
+
+ - A version manager (rvm, nvm, asdf, etc)
+ - A DB (probably Postgres)
+
+__Other useful things todo__
+
+ - Add your ssh key to github ([help link here](https://help.github.com/articles/generating-an-ssh-key/))
+ - Install OpenSSL from homebrew
+  `brew install openssl`
+
+# Tools
+Most tools can be found [here]({{site.baseurl}}/pages/administrative-tooling).
+
+# Misc
+
+- [Holidays]({{site.baseurl}}/pages/holidays).
+- [Conferences and training]({{site.baseurl}}/pages/conferences-and-training).
+- [Sick Pay]({site.baseurl}/pages/sick-pay-entitlement)
+  - If your sick
+    - Stay at home and recover
+    - If you can work then please do
```

Comment:
(As in, if some is really feeling horrible then don't want to pressure them to be working.)